src/util/request-manager.js

```diff
@@ -195,7 +196,6 @@ export default class RequestManager {
 
   _getRequestModule(): RequestModuleT {
     if (!this._requestModule) {
-      const request = require('request');
```

Comment:
This doesn't need to be inlined as we're already inlining all imports via `babel-plugin-transform-inline-imports-commonjs`.